sky/data/data_utils.py

```diff
@@ -126,6 +146,175 @@ def verify_gcs_bucket(name: str) -> bool:
         return False
 
 
+def create_az_client(client_type: str,
+                     **kwargs) -> Client:
+    """Helper method that connects to AZ client for diverse Resources.
+
+    Args:
+      type: str; resource name, e.g. storage, resource, container
+      storage_account_name: str; name of the storage account
+      container_name: str; name of the container.
+
+    Returns:
+      Client object facing AZ Resource of the 'type'.
+    """
+    storage_account_name = kwargs.pop('storage_account_name', None)
+    container_name = kwargs.pop('container_name', None)
+    if client_type == 'container':
+        assert storage_account_name is not None
+        assert container_name is not None
+    subscription_id = azure.get_subscription_id()
+    return azure.get_client(client_type, subscription_id,
+                            storage_account_name=storage_account_name,
+                            container_name=container_name)
+
+
+def verify_az_bucket(storage_account_name: str, container_name: str) -> bool:
+    """Helper method that checks if the AZ Container exists
+
+    Args:
+      name: str; Name of AZ Container.
+
+    Returns:
+      boolean; shows either or not the container exists.
+    """
+    container_client = create_az_client(
+        'container',
```

Comment:
```suggestion
        client_type='container',
```
nit: use kwargs for literal

Reply:
Done.